listenbrainz/webserver/static/js/src/huesound/ColorWheel.tsx

```diff
@@ -37,6 +37,16 @@ type ColorWheelState = {
   centerCircleOpen: boolean;
 };
 
+const defaultProps = {
```

Comment:
I don't even want to talk about this. The interaction between typescript typing and React's defaultProps is awful with optional props with defaults.